internal/grid/connection.go

```diff
@@ -970,7 +969,6 @@ func (c *Connection) handleMessages(ctx context.Context, conn net.Conn) {
 			var m message
 			subID, remain, err := m.parse(msg)
 			if err != nil {
-				logger.LogIf(ctx, fmt.Errorf("ws parse package: %w", err))
```

Comment:
None of these are "startup logs" AFAICT.

You are hiding when a significant system potentially is failing. Of course it is my code, so it will never fail (except the reads-writes).

You are welcome to remove all observability, but it seems important for me to know when disconnects are happening, if I am ever going to debug this.

Reply:
Yes generally for that I think we should convert them into a metrics problem, the issue is right now these logs are generated for simple events like node disconnection. 

While they may serve some debug purpose - it is quite simply very noisy right now. I could keep it all non-network errors must be logged?

Reply:
@harshavardhana I have another PR that separate logs of subsystems. Like replication, peers, grid.. Then we can decide what to hide and what to keep. But overall, I am in favor of having a DEBUG flag that shows verbose log to identify rare issues. I cannot think of another way.

Reply:
something like this: https://github.com/minio/minio/pull/19002

It would be easier to control logging if we use subsystem concept

Reply:
Okay then that will do